source/index.html.md

```diff
@@ -333,38 +333,48 @@ Example response (snippet):
     },
     "processed_at": "2018-07-09T12:58:56+00:00",
     "amount": 50000,
-    "currency": "EUR",
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "currency": "EUR"
 }
 ````
 
 
-## Recurring payments
+## Credential on file
 
-Recurring payments enable you to repeatedly charge cardholders without having
-them to provide card information for subsequent payments.
+Subscriptions are supported using `credential_on_file[]`. There are 2 types of
+subscriptions: `recurring` and `unscheduled` (merchant-initiated) credential on
+file (UCOF).
 
-### Subscription concept
+A `recurring` series consists of transactions with the same amount being charged
+at regular intervals.
 
-Many PSPs have a subscription concept for supporting recurring payments.
-The first approved authorization is the initial recurring authorization (also
-known as "first in series"), all later authorizations are called subsequent
-authorizations. Clearhaus supports subscriptions in the form of recurring
-payments.
+An `unscheduled` credential on file is a series of merchant-initiated
+transactions (MIT) just like a `recurring` series, but with the exception that
+the intervals between withdrawals are irregular; instead, transactions are made
```

Comment:
Not just intervals, also amount? I can't find anything about it in PSD2.

Reply:
Unknown to me.

Reply:
From psd2-sca-for-remote-electronic-transaction-implementation-guide.pdf (VISA) p. 46
![Screenshot from 2019-08-15 11-29-02](https://user-images.githubusercontent.com/24522411/63086421-1d37dd00-bf50-11e9-8cd0-73f3dba58203.png)

Reply:
Clarification from the Danish NCA seem to suggest that varying amount for both recurring and UCOF is perfectly fine — as long as the change in amount that the merchant is withdrawing is covered by the agreement between merchant and cardholder.